cli/bench/space.go

```diff
@@ -54,3 +57,36 @@ func dropBenchmarkSpace(tarantoolConnection *tarantool.Connection) error {
 	}
 	return nil
 }
+
+// fillBenchmarkSpace fills benchmark space
+// with a PreFillingCount number of records
+// using connectionPool for fast filling.
+func fillBenchmarkSpace(ctx context.BenchCtx, connectionPool []*tarantool.Connection) int {
+	var insertMutex sync.Mutex
+	var waitGroup sync.WaitGroup
+	filledCount := 0
+	for i := 0; i < ctx.Connections; i++ {
+		waitGroup.Add(1)
+		go func(tarantoolConnection *tarantool.Connection) {
+			defer waitGroup.Done()
+			for filledCount < ctx.PreFillingCount {
+				insertMutex.Lock()
+				if filledCount == ctx.PreFillingCount {
```

Comment:
Wow, I understand that you are checking if this is an "extra" iteration, but it's not that obvious. Please add a comment.

Reply:
added